app/components/widgets/forms/InlineEditingInput.scss

```diff
@@ -60,67 +59,20 @@
   }
 }
 
-.component {
-  position: relative;
+:global(.YoroiClassic) .component {
+  margin-bottom: 10px;
+}
 
+:global(.YoroiModern) .component {
   .disabled {
     background-color: transparent;
-    border-color: var(--theme-input-border-color);
-    color: var(--theme-input-text-color);
-  }
-
-  &.inactive {
-    &:hover {
-      input {
-        cursor: pointer;
-      }
-    }
   }
 
-  // need this as Firefox disabled bubbling of click events on disabled inputs
-  input[disabled] {pointer-events:none}
-
   .button {
```

Comment:
I noticed for this file you just deleted a bunch of css. Is this okay?

Reply:
@SebastienGllmt 

Those were common in both of Themes.
Logically those are sill there but at common place and used in both of the Themes.
![image](https://user-images.githubusercontent.com/19986226/57904925-b6068000-78af-11e9-8a22-887b71d05f61.png)